src/v2/Apps/Auction/Components/AuctionActiveBids.tsx

```diff
@@ -236,7 +237,7 @@ const BidButton: React.FC<
         router.push(href)
       }}
     >
-      Increase Bid
+      {lotStanding.saleArtwork?.endedAt ? "Bidding closed" : "Increase Bid"}
```

Comment:
We'll want to update the [`AuctionActiveBids` test](src/v2/Apps/Auction/Components/__tests__/AuctionActiveBids.jest.tsx) to reflect this new behavior.

Reply:
You also may want to check with design. From the screenshots it looks like the two buttons are slightly different sizes based on which text is displaced which is slightly awkward. Also maybe it's odd to have "Bidding closed" displayed on a button as it's not really a CTA?

Reply:
I added a test